caiman/tests/test_temporal.py

```diff
@@ -10,12 +10,7 @@ def test_make_G_matrix():
     T = 6
     G = cnmf.temporal.make_G_matrix(T, g)
     G = G.todense()
-    true_G = np.matrix(
-        [[1., 0., 0., 0., 0., 0.],
-         [-1., 1., 0., 0., 0., 0.],
-         [-2., -1., 1., 0., 0., 0.],
-         [-3., -2., -1., 1., 0., 0.],
-         [0., -3., -2., -1., 1., 0.],
-         [0., 0., -3., -2., -1., 1.]])
+    true_G = np.matrix([[1., 0., 0., 0., 0., 0.], [-1., 1., 0., 0., 0., 0.], [-2., -1., 1., 0., 0., 0.],
```

Comment:
older is easier to read here given it shows the matrix.

Reply:
ok, will revert and prevent it from doing this again.